### PR TITLE
change labels when external visibility disabled

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: mc2/controllers/docker/forms.py [hunk 1]
```diff
@@ -15,12 +15,23 @@ class DockerControllerForm(ControllerForm):
...
+            'placeholder': '(optional)'}),
+        required=False)
     domain_urls = forms.CharField(
         widget=forms.TextInput(attrs={
             'class': 'form-control',
             'placeholder': '(optional)'}),
         required=False)
+    external_visibility = forms.BooleanField(
+        required=False,
+        initial=True,
```
Comment (jerith) [inline mc2/controllers/docker/forms.py]: Do we want this to default to true?

File: mc2/controllers/docker/models.py [hunk 2]
```diff
@@ -62,11 +63,16 @@ def get_marathon_app_data(self):
         service_labels = self.get_default_app_labels()
         service_labels.update({
             "domain": domains,
```
Comment (jerith) [inline mc2/controllers/docker/models.py]: This is used by the consul-template+nginx setup, so it should probably move to the "external visibility" section.

File: mc2/controllers/docker/forms.py (hunk 1 above)
Comment (miltontony) [inline mc2/controllers/docker/forms.py]: I'm happy to change this to default to false.

**Comment by miltontony** [comment]
@jerith ready for re-review

**Comment by saeedmb8** [comment]
👍🏻

**Pull request merged**